docs/pywbemcli/commands.rst

```diff
@@ -1534,6 +1549,412 @@ See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
 command.
 
 
+.. index:: pair: command groups;statistics command group
+
+.. _`Statistics command group`:
+
+``statistics`` command group
+----------------------------
+
+The ``statistics`` command group includes commands that manage and display
+statistics on the WBEM server operations executed by pywbemcli.
+
+.. index:: pair: Operation statistics;statistics commands
+<<<<<<< HEAD
```

Comment:
The file contains multiple merge conflicts. Please resolve them.

Reply:
Sure does.  If I could explain that one I would.  But in any case, cleaned up.  Some times they were 3 levels deep.  

These are the paragraphs that really merit reading in detail since they are the one that explain statisticss